src/components/PrimaryPanes/tests/SourcesTree.spec.js

```diff
@@ -155,20 +155,6 @@ describe("SourcesTree", () => {
       });
     });
 
-    describe("updates list items", () => {
```

Comment:
Nixed this test because we aren't doing `selectItem` here anymore -- we're doing it in the action.